x11-misc/revelation/revelation-0.5.3.ebuild

```
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_8 )
+
+inherit python-single-r1 autotools gnome2
+
+DESCRIPTION="A password manager for GNOME"
+HOMEPAGE="https://revelation.olasagasti.info/ https://github.com/mikelolasagasti/revelation"
+SRC_URI="https://github.com/mikelolasagasti/revelation/releases/download/${P}/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
```

```suggestion
KEYWORDS="~amd64 ~x86"
```
With build system change, let's drop any old exotic arches.